### DISCUSSION TITLE
Error during grain reconstruction

### DISCUSSION THREAD

**Discussion by Ericpan3** [mtex-toolbox/mtex#1731] [Ask Anything]
I'm trying to run a series code to smooth and reconstruct the grains while there are errors. The same code ran before. Not sure what happened here.  Is there something wrong with my code or MATLAB?
![2023-02-06 (2)](https://user-images.githubusercontent.com/80721968/217124442-202efcf8-3d00-4ebf-8422-3c747811dd1c.png)
![2023-02-06 (3)](https://user-images.githubusercontent.com/80721968/217124541-d8a80b3d-6b87-4ef6-9c82-c0ce1ad84074.png)

**Comment by kilir** [comment]
Hi,
the error message says you do not have a phase called FCC.
Cheers, Rüdiger

**Reply by Ericpan3** [reply]
Thank you Rudiger,
May I ask how should I import FCC phase here? When I import my EBSD file, it shows:
%% Specify Crystal and Specimen Symmetries

% crystal symmetry
CS = {... 
  'notIndexed',...
  crystalSymmetry('432', [4 4 4], 'mineral', 'FCC', 'color', [0.53 0.81 0.98])};

% plotting convention
setMTEXpref('xAxisDirection','east');
setMTEXpref('zAxisDirection','intoPlane');

**Comment by lfgmorales** [comment]
Hi

CS reads only which phases you added when you ran your ebsd map, and apparently you asked your EBSD system to measure FCC. However it seems you don’t have anything indexed as FCC, so it doesn’t matter if you have fcc listed there. What does ebsd returns as pixels and percentage?

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 7 Feb 2023, at 14:44, Ericpan3 ***@***.***> wrote:

﻿

Thank you Rudiger,
May I ask how should I import FCC phase here? When I import my EBSD file, it shows:
%% Specify Crystal and Specimen Symmetries

% crystal symmetry
CS = {...
'notIndexed',...
crystalSymmetry('432', [4 4 4], 'mineral', 'FCC', 'color', [0.53 0.81 0.98])};

% plotting convention
setMTEXpref('xAxisDirection','east');
setMTEXpref('zAxisDirection','intoPlane');

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1731#discussioncomment-4893702>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5INZTDBHS7KBXNUKQXR3WWJGS3ANCNFSM6AAAAAAUTLQACU>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1731#discussioncomment-4893702", "url": "https://github.com/mtex-toolbox/mtex/discussions/1731#discussioncomment-4893702", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by Ericpan3** [reply]
Thank you Luiz,

I got it now. I changed all "FCC" to "Aluminum", and the code works.
